src/datasource/components/Logs/LogsCell.tsx

```diff
@@ -0,0 +1,325 @@
+            </span>
+            <table>
+                <tr>
+                    <th>
+                        Field
+                    </th>
+                    <th>
+                        Value
+                    </th>
+
+                </tr>
+                {
+                    Array.from(log.keys()).map((key) => (
+                        <ExpandedLogKeyVal
+                            field={key}
+                            val={log.get(key)}
+                            key={key}
+                        />
+                    ))       
+                }
+            </table>
+            {
+                // TODO: Should this value be configurable?
+                log.has('trace_id') && datasourceName && datasourceUid ? 
+                (
+                        <a href={formattedLink}>
+                            <Button
+                                size={'sm'}
+                                variant={'primary'}
+                            >
+                                View in {datasourceName}
+                            </Button>
+                        </a>
+                )
+                    : ''
+            }
+            
+    </div>
+    );
+}
+
+
+const DocumentCell = (log: Log, style: any, rowIndex: number, expanded: boolean, datasourceUid: string, datasourceName: string) => (
+    <div
+        style={{
+                display: 'inline-block',
+                lineHeight: '2em',
+                fontFamily: 'monospace',
+                fontSize: '12px',
+                overflow: 'hidden',
+                paddingTop: '10px',
+                ...style
+            }}
+    >
+        <div style={{maxHeight: '115px', overflow: 'hidden'}}>
+            {
+                Array.from(log.keys()).map((key) => (
+                    <LogKeyVal
+                        field={key}
+                        val={log.get(key)}
+                        key={key}
+                    />
+                ))
+            }
+        </div>
+        {
+            expanded ?
+                (<ExpandedDocument
+                    log={log}
+                    index={rowIndex}
+                    datasourceUid={datasourceUid}
+                    datasourceName={datasourceName}
+                />)
+                      
+            : ''
+        }
+    </div>
+)
+
+const TimestampCell = (timestamp: number, style: any, rowIndex: number, expandedRows: boolean[], onClick: ((index: number) => void)) => {
+    const getFoldIcon = () => {
+        if (expandedRows[rowIndex]) {
+            return 'angle-down';
+        }
+        return 'angle-right';
+    };
+
+    const userTimezone = getTimeZone();
+    let formattedTime: string | null = null;
+    if (timestamp) {
+        let datetime = DateTime.fromMillis(timestamp);
+        if (userTimezone !== 'browser') {
+            datetime = datetime.setZone(userTimezone);
+        }
+        formattedTime = datetime.toISO();
```

Comment:
Does Grafana not provide a common util for rendering datetimes? Then we wouldn't need to import the extra dependency for `luxon`

Reply:
The date format is not in a humanized form:
![Screenshot 2024-01-12 at 11 34 22 AM](https://github.com/slackhq/slack-kaldb-app/assets/771133/b9dc5737-bba4-4265-a192-9c57b076115e)

![Screenshot 2024-01-12 at 11 34 31 AM](https://github.com/slackhq/slack-kaldb-app/assets/771133/e4208585-902a-461b-9b5b-d36b18337e49)